deployments/apiv2/services/users/src/main.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+import asyncpg
+import json
+import jwt
+import uuid
+
+from datetime import datetime, timedelta, timezone
+from calendar import timegm
+
+from argon2 import PasswordHasher
+from argon2.exceptions import VerifyMismatchError, InvalidHash
+
+from fastapi import FastAPI, Request, Depends, HTTPException, status
+from fastapi.responses import JSONResponse
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+
+from core.config import JWT_SECRET_KEY, JWT_AUDIENCE, JWT_ALGORITHM, ACCESS_TOKEN_EXPIRE_MINUTES
+from core.db import Database
+from core.auth import ProtectedAny
+from models.users import UserLogin, UserCreate, UserProfile
+from models.token import JWTMeta, JWTDetails, JWTPayload, AccessToken
+from models.errors import LoginError, RegistrationError
+
+db = Database()
+app = FastAPI()
+
+
+@app.middleware("http")
+async def db_session_middleware(request: Request, call_next):
+    request.state.pgpool = db.pool
+    response = await call_next(request)
+    return response
+
+
+@app.on_event("startup")
+async def startup():
+    await db.create_pool()
+
+
+@app.on_event("shutdown")
+async def shutdown():
+    await db.close()
+
+
+@app.get("/users/me", response_model=UserProfile)
+async def index(request: Request, token=Depends(ProtectedAny(scope=["users:free"]))):
+    try:
+        async with request.state.pgpool.acquire() as con:
+            user_id = uuid.UUID(hex=token["userid"])
+            rows = await con.fetchrow("select name, email, user_id, account_type, created_at, updated_at, data->'scope' as scope from users where user_id = $1", user_id)
+
```

Comment:
whats the data->'scope' as scope?

Reply:
`data` is a `jsonb` column in the table, so `data->'scope'` is getting the `scope` field from the json object for the `data` column in that row, the `as scope` is setting the output column name to `scope` instead of the something like `?column?` that would be set otherwise